style.css

```diff
@@ -148,6 +148,427 @@ body {
+}
+
+.middle-section__form-colors {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  width: 133px;
+  height: 30px;
+  padding: 0;
+  margin-top: 5px;
+}
+
+.form__colors-btn {
+  display: none;
+}
+
+.form__colors-item {
+  position: relative;
+  display: inline-block;
+  width: 29px;
+  height: 29px;
+}
+
+.form__colors-item::before {
+  content: "";
+  position: absolute;
+  left: 3px;
+  top: 0;
+  display: block;
+  width: 23px;
+  height: 23px;
+  background: #dedede;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.form__colors-item:nth-of-type(2)::before {
+  background: #f29f9f;
+}
+
+.form__colors-item:nth-of-type(3)::before {
+  background: #a8bff9;
+}
+
+.form__colors-item:nth-of-type(4)::before {
+  background: #90bd8f;
+}
+
+input[type="radio"]:checked + .form__colors-item::before {
+  left: 0;
+  width: 29px;
+  height: 29px;
+  -webkit-transition: all 300ms ease-in-out;
+  -o-transition: all 300ms ease-in-out;
+  transition: all 300ms ease-in-out;
+  -webkit-transform: translate(0, -9%);
+  -ms-transform: translate(0, -9%);
+  transform: translate(0, -9%);
+}
+
+.middle-section__colors-and-price {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  margin: 0;
+  height: 46px;
+  min-width: 250px;
+  margin-bottom: 22px;
+}
+
+.middle-section__order {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding: 0;
+}
+
+.middle-section__order-item {
+  width: 256px;
+  padding-top: 9px;
+  padding-bottom: 26px;
+  margin-bottom: 10px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  text-align: center;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  cursor: pointer;
+  border: 1.5px solid #000;
+}
+
+.middle-section__price {
+  margin: 0;
+  margin-right: 8px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.middle-section__subtitle {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 10px;
+  letter-spacing: .07em;
+  color: #000;
+  opacity: .5;
+}
+
+button {
+  background: none;
+}
+
+.middle-section__btn,
+.middle-section__btn_other {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+.middle-section__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 20px;
+  color: #000;
+}
+
+.middle-section__desription {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 9px;
+  line-height: 13px;
+  color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)